### DISCUSSION TITLE
When hovering over the (0,0) position on a grid the item is not highlighted

### DISCUSSION THREAD

**Discussion by MaiveBel** [peter-kish/gloot#202] [Q&A]
When I hover oven an item at the 0,0 position on the grid it is not highlighted 
![image](https://github.com/peter-kish/gloot/assets/48601667/6528ddd3-5082-4e6b-bea6-bbf73006e51b)
(mouse at 0,0)

![image](https://github.com/peter-kish/gloot/assets/48601667/5b21cd2b-e341-40d1-af82-a2167d94debb)
(mouse at 1,0)

However any other other slot is fine, this makes 1x1 items inaccessible to the player if placed in the 0,0 slot, while being a major hassle for any larger item.

You cant drag items into the 0,0 slot, but you can add them in and they are stuck there permanently and unintractable with by the player

**Comment by peter-kish** [comment]
That's weird... I had no problem hovering/grabbing items at (0,0) so far:
![0_0_hover](https://github.com/peter-kish/gloot/assets/11254996/3d325d16-cd96-4b64-b270-45e686cba24d)

Did you try doing the same in the example scenes?
Also, can you give some information about your scene? Maybe some transparent control is positioned on top of the (0,0) field and the blocking mouse input.
